app/models/customer.rb

```diff
@@ -0,0 +1,14 @@
+class Customer < ApplicationRecord
+    
+    validates :first_name, presence: true, format: { with: /\A[a-z\-' ]+\z/i }
+    validates :last_name, presence: true, format: { with: /\A[a-z\-' ]+\z/i }
+    validates :phone, presence: true
+    validates :phone, numericality: { only_integer: true }
+    validates :phone, length: { is: 10 }
+    validates :email, presence: true, format: { with: /\A[a-z\-' ]+\z/i }
```

Comment:
You want to use the email validation that's shown on the lesson, not copy the first/last name. Names and email addresses are quite different, or at least I haven't seen anyone with a real name that has an `@` or `.com` in it.

```suggestion
    validates :email, presence: true, email: true
```